product_docs/docs/eprs/6.2/08_xdb_cli/03_xdb_cli_commands/25_set_pdn_node.mdx

```diff
@@ -30,7 +30,7 @@ Examples
 The following example sets the primary node with publication database `ID 9` as the primary definition node.
 
 ```text
-$ java -jar edb-repcli.jar -setasPDN 9 -repsvrfile ~/pubsvrfile.prop
-Updating the database node to be promoted as the new PDN node.
-The database has been successfully promoted as the new PDN node.
+$ java -jar edb-repcli.jar -setasMDN 9 -repsvrfile ~/pubsvrfile.prop
```

Comment:
Please change the command to 'setasmdn'. Rep cli commands are all lower case for simplicity.